tests/cluster_tests/RavenRunsRaven/test_raven_running_raven_int_models.xml

```diff
@@ -93,10 +92,6 @@
     </MonteCarlo>
   </Samplers>
 
-  <!-- DATABASES -->
-  <Databases>
-    <HDF5 name="test_external_db" readMode="overwrite"/>
-  </Databases>
```

Comment:
TestInfo should be updated, also check other modified tests

Reply:
I do not think it is a modification worth to be mentioned in the revision history since it does not change any functionality or the purpose of the test... 
If you want me to modify it, I can...let me know.

Reply:
For me it is ok.